packages/carequality-sdk/src/models/bundle.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod";
+import { identifierSchema, meta, objectNumericValue, objectValue } from "./shared";
+import { organizationSchema } from "./organization";
+
+export const bundleSchema = z.object({
+  xmlns: z.string(),
```

Comment:
Are we going to read this? If not, then I suggest we don't map it on the schema one less thing to fail when parsing.

Same applies to all other schemas and properties:
- if we don't use it, let's avoid mapping it;
- if we use it, make sure it's required, otherwise let's make it optional if we're not sure.

Just bringing it up b/c it took us a few iterations to get schemas right when integrating w/ external systems. 😅